pytorch_lightning/accelerators/ddp2_accelerator.py

```diff
@@ -218,3 +218,6 @@ def sync_tensor(self,
                     group: Optional[Any] = None,
                     reduce_op: Optional[Union[ReduceOp, str]] = None) -> torch.Tensor:
         return sync_ddp_if_available(tensor, group, reduce_op)
+
+    def reference_model(self, model) -> LightningModule:
```

Comment:
I guess we talke about...
```suggestion
    def get_reference_model(self, model) -> LightningModule:
```